maven-enforcer-plugin/src/main/java/org/apache/maven/plugins/enforcer/EnforceMojo.java

```diff
@@ -127,10 +127,12 @@ public class EnforceMojo extends AbstractMojo {
     private PlexusConfiguration rules;
 
     /**
-     * List of strings that matches the EnforcerRules to execute.
+     * List of strings that matches the EnforcerRules to execute. Replacement for the <code>rules</code> property.
+     *
+     * @since 3.2.0
      */
-    @Parameter(required = false, property = "rules")
-    private List<String> commandLineRules;
+    @Parameter(required = false, property = "enforcer.rules")
```

Comment:
It seemed to me it would be more consistent to have it on the field, but I'll move it to the setter then.

Reply:
about consistent I thought that once rulesToExecute is set from setter with annotation and then from setter without annotation.

but of course it can be a my subjective opinion

Reply:
Yeah, I thought about is from the other fields (with `@Parameter`) point of view. And the other setter is just a glue to support the deprecated parameter. But it does not really matter that much. I moved the annotation to the setter.